app/helpers/view_helper.rb

```diff
@@ -62,7 +62,8 @@ def enrichment_error_url(provider_code:, course:, field:, message: nil)
         age_range_in_years: "#{base}/age-range?display_errors=true",
         sites: "#{base}/schools?display_errors=true",
         study_sites: (course.provider&.study_sites&.none? ? "#{provider_base}/study-sites" : "#{base}/study-sites").to_s,
-        accrediting_provider: accredited_provider_publish_provider_recruitment_cycle_course_path(course.provider_code, course.recruitment_cycle_year, course.course_code)
+        accrediting_provider: accredited_provider_publish_provider_recruitment_cycle_course_path(course.provider_code, course.recruitment_cycle_year, course.course_code),
+        applications_open_from: "#{base}/applications-open"
```

Comment:
Seeing this whole helper file, makes me wondered why not use rails route helpers? Is there a specific reason?

Reply:
It was done like this when the project was in 2 repos, the URL was generated from the API side. It's currently functioning as a shim. 

We can make a card to change it.